javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,14 @@
+/* exported oddOrEven */
+
+function oddOrEven(numbers) {
+  var array = [];
+  for (var i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      array.push('even');
+    } else if (numbers[i] % 2 === 1) {
```

Comment:
This could be done with just an else, because you are checking for the opposite of the first conditional check.